app/logic/displayName.py

```diff
@@ -0,0 +1,17 @@
+from app.models import * 
+
+def displayName(name, abbreviation):
```

Comment:
This function needs a better name. The name should give you enough information that when you encounter it in code you have an understanding of what it does and how to use it without needing to look up the documentation.

Should be an action name, and should mention course.

Reply:
We have named it displaycourse and createcoursedisplayname as a function name